frontend/src/k8sTypes.ts

```diff
@@ -858,3 +858,57 @@ export type DataScienceClusterKindStatus = {
   installedComponents: { [key in StackComponent]?: boolean };
   phase?: string;
 };
+
```

Comment:
We dont want edge specific pipeline run types. they are not needed.

and make sure when creating types for pipelines, that they are following the types here: https://tekton.dev/docs/pipelines/pipeline-api/#tekton.dev/v1beta1.PipelineRun

keep the naming and everything consistent because that is the source